book/09-git-and-other-scms/sections/client-tfs.asc

```diff
@@ -328,8 +329,8 @@ PS> git log --all --oneline --graph --decorate
 * b75da1a New project
 ----
 
-Now we're ready to complete our contribution by checking in our code to the TFVC server.
-We'll use the `rcheckin` command here to create a TFVC changeset for each Git commit in the path from HEAD to the first `tfs` remote branch found (the `checkin` command would only create one changeset, sort of like squashing Git commits).
+Ngayon tayo ay handa nang kumpletuhin ang ating pag-ambag sa pamamagitan ng pagtsi-check in ng ating code sa TFVC na server.
+Gagamitin natin ang `rcheckin` na utos dito upang lumikha ng isang TFVC na hanay ng pagbabago para sa bawat Git na commit sa landas mula sa HEAD patungo sa unang `tfs` na remote na branch na natagpuan (ang `checkin` na utos ay lilikha lamang ng isang hanay ng pagbabago, tulad ng pagpipi ng Git na mga commit).
```

Comment:
lilikha  to lumikha

Reply:
will retain the phrase `lilikha lamang ng isang`, to emphasize future tense equivalent for `would only create`